src/backend/schemas/langchain_chat.py

```diff
@@ -1,23 +1,9 @@
-from typing import List
-
-from pydantic import Field
-
 from backend.schemas.chat import BaseChatRequest
-from backend.schemas.tool import Tool
 
 
 class LangchainChatRequest(BaseChatRequest):
     """
     Request shape for Langchain Streamed Chat.
-    See: https://github.com/cohere-ai/cohere-python/blob/main/src/cohere/base_client.py#L1629
     """
 
-    preamble: str | None = Field(
-        default=None,
-        title="A string to override the preamble.",
-    )
-    tools: List[Tool] = Field(
-        default_factory=list,
-        title="List of managed tools to use for the response.",
-        exclude=True,
-    )
+    pass
```

Comment:
hmmm whats the point of this? just to differentiate req types for diff chat reqs?

Reply:
yeah, there's a lot of fields on cohere chat request which are not required. Better to only pass around what is needed IMO